src/grafana/values/values.yaml

```diff
@@ -5,3 +5,30 @@ sidecar:
   datasources:
     enabled: true
     label: grafana_datasource
+
+extraSecretMounts:
+  - name: auth-generic-oauth-secret-mount
+    secretName: sso-client-uds-core-admin-grafana
+    defaultMode: 0440
+    mountPath: /etc/secrets/auth_generic_oauth
+    readOnly: true
+
+grafana.ini:
+  server:
+    root_url: https://grafana.admin.###ZARF_VAR_DOMAIN###
+  auth.generic_oauth:
+    # https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/keycloak/
+    enabled: true
+    client_id: $__file{/etc/secrets/auth_generic_oauth/clientId}
+    client_secret: $__file{/etc/secrets/auth_generic_oauth/secret}
+    scopes: openid profile
+    email_attribute_path: email
+    login_attribute_path: preferred_username
+    name_attribute_path: name
+    name: UDS Identity Service
+    auth_url: https://keycloak.admin.###ZARF_VAR_DOMAIN###/realms/uds/protocol/openid-connect/auth
```

Comment:
Curious on the why here for using the `keycloak.admin` VS instead of the `sso` VS? Tracking that grafana is on the admin gateway so it sorta makes sense, just not sure if that makes the user experience weird when the admin VS is typically used for admin/master realm things?

Reply:
I guess maybe it doesnt matter? My thinking was if grafana is on the admin GW (which could be an isolated network), it might be a problem if egress on that admin network was limited

Reply:
Yeah that's a good callout - good with this and we can see if it is problematic down the road.